snapcraft/plugins/python2.py

```diff
@@ -67,6 +67,7 @@ def __init__(self, name, options):
         super().__init__(name, options)
         self.stage_packages.extend([
             'python-dev',
+            'python-minimal',
```

Comment:
It indeed might be, I pushed it here to see what happened on travis; I'm not sure how to fix this one since `run` is called from the class which generates an env by calling env.